scipy/stats/tests/test_stats.py

```diff
@@ -7031,7 +7027,7 @@ def test_nan_policy_raise(self):
             stats.alexandergovern(*args, nan_policy='raise')
 
     def test_nan_policy_omit(self):
-        args_nan = [[1, 2, 3, None, 4], [1, np.nan, 19, 25]]
```

Comment:
I don't know why `None` inputs were supported in the first place (probably pandas dfs with no values) but I think it's OK to break this. Let me know what you think